hermes/cmd/hermes.go

```diff
@@ -8,9 +8,9 @@ const (
 	flagConfig = "config"
 )
 
-// NewRelayer creates a new relayer command that holds
+// NewHermes creates a new Hermes relayer command that holds
 // some other sub commands related to hermes relayer.
-func NewRelayer() *cobra.Command {
+func NewHermes() *cobra.Command {
```

Comment:
Maybe create an alias for not breaking the API?

Reply:
I changed the name to `NewHermes` because the command is called "hermes".

It is attached to the `ignite relayer` "namespace" to allow other relayers, so `NewHermes` is called when `ignite relayer hermes` is executed. Maybe in the future we might have something like `ignite relayer go` for example. So "relayer" is a namespace for IBC relayers while "hermes" is the name of the actual command.

I'm not sure what you mean by breaking the API